### ISSUE TITLE
config.properties

### ISSUE THREAD

**Issue by BattleHook** [Zweistein2/AgrarMonitor#1]
Hello, 

I really like what you did with AgrarMonitor.

when running the Agrarmonitor.exe on PC, where can I find the "config.properties" ?

**Comment by BattleHook** [comment]
my savegames are located in E:\Documents\My Games\FarmingSimulator2022
I edited config.properties as follows:
savegameDir=E:/Documents/My\ Games/FarmingSimulator2022
but it does nor show the data.

**Comment by Zweistein2** [comment]
Try

savegameDir=E:\\\Documents\\\My\ Games\\\FarmingSimulator2022

then it should work

**Comment by BattleHook** [comment]
OK, I'll try that.
but: you can use '/' (as in Linux) in paths since Windows XP, so why should I use \ which has to be escaped (\\) ?
You can use \\ or / but / is better because it is OS-independent. Internally, Java will convert it to the file seperator of the OS

**Comment by Zweistein2** [comment]
Yes, normally you could use a path with '/' instead of '\\\'. Both work fine in Java / with AgrarMonitor, but sometimes '/' seems to be a bit buggy.

Did you check if the path is correct? On my computer the "documents"-folder is inside my user-home. So the path should look more like "E:\\Users\\NAME\\Documents..."

If the path is correct, you can verify that AgrarMonitor is trying to load the correct savegame. By default, the application tries to load the first savegame slot. If there is no savegame in slot 1, no data is shown. To switch to the correct savegame you should edit the last part of the URL in your browser: "?savegame=X" where X is the number of the slot you want to load. (In the next version there will be a button to switch savegames, so no URL-editing will be necessary anymore)

If it still doesn't work, could you send the error messages that appear on the command line?
There should be some sort of "java.io.FileNotFoundException"-Exceptions.

**Comment by BattleHook** [comment]
commandline? I run it by double-clicking the AgrarMonitor.jar file in the Windows Explorer.

**Comment by Zweistein2** [comment]
You could open a command line and start it with "java -jar AgrarMonitor.jar". Then you can see the (error) log

**Comment by BattleHook** [comment]
Thanks: now it works with: savegameDir=E:\\Documents\\My\ Games\\FarmingSimulator2022

and I made a batch file with : 
`if not "%minimized%"=="" goto :minimized
set minimized=true
start /min cmd /C "%~dpnx0"
goto :EOF
:minimized
java -jar AgrarMonitor.jar /MIN`

Furthermore can you make it so that in config.properties I can pass language and standard savegame to load ?
